mdx-layout-core/src/NavLink.tsx

```diff
@@ -1,42 +1,36 @@
 import type { Page } from '@divriots/studio-doc-compiler';
 import React from 'react';
-import path from 'path';
 import { styles } from './NavGroup';
 
-const relativeUrl = (url: string) =>
-  path.join(
-    Array(url.split('/').length - 1)
-      .fill('..')
-      .join('/'),
-    url
-  );
-
-export const NavLink = ({ page }: { page: Page }) => {
+export const NavLink = ({ base, page }: { base: string; page: Page }) => {
+  if (!page.url) {
+    return null;
+  }
   const [hover, setHover] = React.useState(false);
+  const pageUrlWithoutOrigin = base + page.url;
```

Comment:
what if base is null or undefined ?

Reply:
it's a `string`, so if it's not defined during the docs export, it should be set to `/`